bundle/edu.gemini.p2checker/src/test/scala/edu/gemini/p2checker/rules/NonSiderealTargetRulesSpec.scala

```diff
@@ -132,7 +132,6 @@ class NonSiderealTargetRulesSpec extends RuleSpec {
       }
       e.setDataObject(toc)
     }
-    f.createObsComponent(p, inst, null) <| { i => o.addObsComponent(i) }
```

Comment:
The equivalent line in `RuleSpec.scala:30` (the next file down) remains … is this because we have an instrument here but not in `RuleSpec`?